### PR TITLE
Add --pretrained_model_name_revision option to train_dreambooth.py

### PR DESCRIPTION
**Pull request by shirayu**
This PR enables specify the revision of the pretrained model

### CODE REVIEW

File: examples/dreambooth/train_dreambooth.py [hunk 1]
```diff
@@ -35,6 +35,13 @@ def parse_args():
         required=True,
         help="Path to pretrained model or model identifier from huggingface.co/models.",
     )
+    parser.add_argument(
+        "--pretrained_model_name_revision",
```
Comment (patrickvonplaten) [inline examples/dreambooth/train_dreambooth.py:39]: wonder whether we should call this just `revision` @patil-suraj what do you think?

Comment (patil-suraj) [reply examples/dreambooth/train_dreambooth.py:39]: Agree, let's just call this `revision`

**Comment by shirayu** [comment]
Thank you for your comments!
I’m with you.
I renamed them!

**Comment by patrickvonplaten** [comment]
Thanks a lot for your PR @shirayu :heart:

**Pull request merged**